### ISSUE TITLE
GraphProfiler wrong trace timeline on imagenet-resnet examples

### ISSUE THREAD

**Issue by yogurfrul** [tensorpack/tensorpack#819]
I add GraphProfiler() to the callback in examples/ResNet/imagenet-resnet.py script
then run as below
`./imagenet-resnet.py --data path/to/data --gpu 3,4 --batch 64`
when I put the chrome-trace-50.json to chrome://tracing/ ,
the timeline duration are so many years!!!
I found this in chrome-trace-50.json , normal nodestats ts are all startwith 15xxx while some nodestats ts are startwith 19xxx, that's why the timeline duration are many year!
```
        {
            "args": {
                "op": "MEMCPYDtoH",
                "name": "edge_7572_tower0/linear/BiasAdd"
            },
            "tid": 0,
            "name": "MEMCPYDtoH",
            "ph": "X",
            "pid": 5,
            "dur": 359,
            "ts": 19978045970716470,
            "cat": "Op"
        },
        {
            "args": {
                "op": "MEMCPYDtoH",
                "name": "edge_1079_tower0/gradients/tower0/gap/output_grad/DynamicStitch"
            },
            "tid": 1,
            "name": "MEMCPYDtoH",
            "ph": "X",
            "pid": 5,
            "dur": 2,
            "ts": 1531302016698591,
            "cat": "Op"
        },
……………………
        {
            "args": {
                "op": "Sub",
                "name": "tower0/group2/block0/conv3/bn/AssignMovingAvg/sub_1"
            },
            "tid": 0,
            "name": "Sub",
            "ph": "X",
            "pid": 17,
            "dur": 4,
            "ts": 19978045970621550,
            "cat": "Op"
        },
        {
            "args": {
                "op": "Sub",
                "name": "tower0/group2/block0/conv3/bn/AssignMovingAvg_1/sub_1"
            },
            "tid": 0,
            "name": "Sub",
            "ph": "X",
            "pid": 17,
            "dur": 67,
            "ts": 19978045970621559,
            "cat": "Op"
        },
```

**Comment by ppwwyyxx** [comment]
Not a tensorpack issue..
Is this reproducible?

**Comment by yogurfrul** [comment]
yes ,it can be reproducible.
the problem  happened in one step of about 40~50 step,
the trace of step before it  are normal, after it the trace are abnormal.

**Comment by ppwwyyxx** [comment]
Could you post more information following the issue template? If I could reproduce your problem I can file an issue to tensorflow. Otherwise there is nothing I can do.

**Comment by yogurfrul** [comment]
For any unexpected problems, __PLEASE ALWAYS INCLUDE__:
1. What you did:
	+ If you're using examples:
        ` examples/ResNet/imagenet-resnet.py `
		+ What's the command you run:
        ` ./imagenet-resnet.py --data path/to/data --gpu 3,4 --batch 64 `
		+ Have you made any changes to code? Paste them if any:
        just add GraphProfiler to callbacks in line 76 :
        ``` 
        callbacks = [
            ModelSaver(),
            EstimatedTimeLeft(),
            ScheduledHyperParamSetter(
                'learning_rate', [
                    (0, min(START_LR, BASE_LR)), (30, BASE_LR * 1e-1), (60, BASE_LR * 1e-2),
                    (90, BASE_LR * 1e-3), (100, BASE_LR * 1e-4)]),
            **GraphProfiler()**
        ]
        ```
    https://github.com/tensorpack/tensorpack/blob/master/examples/ResNet/imagenet-resnet.py#L76
2. What you observed, including but not limited to the __entire__ logs.
    
    I found this in chrome-trace-50.json, normal nodestats start timestamp are all startwith 15xxx while some nodestats start timestamp are startwith 19xxx,the timeline duration are so many year!
    '''
           {
                "args": {
                    "op": "MEMCPYDtoH",
                    "name": "edge_7572_tower0/linear/BiasAdd"
                },
                "tid": 0,
                "name": "MEMCPYDtoH",
                "ph": "X",
                "pid": 5,
                "dur": 359,
                "ts": 19978045970716470,
                "cat": "Op"
            },
            {
                "args": {
                    "op": "MEMCPYDtoH",
                    "name": "edge_1079_tower0/gradients/tower0/gap/output_grad/DynamicStitch"
                },
                "tid": 1,
                "name": "MEMCPYDtoH",
                "ph": "X",
                "pid": 5,
                "dur": 2,
                "ts": 1531302016698591,
                "cat": "Op"
            },
            {
                "args": {
                    "op": "Sub",
                    "name": "tower0/group2/block0/conv3/bn/AssignMovingAvg/sub_1"
                },
                "tid": 0,
                "name": "Sub",
                "ph": "X",
                "pid": 17,
                "dur": 4,
                "ts": 19978045970621550,
                "cat": "Op"
            },
            {
                "args": {
                    "op": "Sub",
                    "name": "tower0/group2/block0/conv3/bn/AssignMovingAvg_1/sub_1"
                },
                "tid": 0,
                "name": "Sub",
                "ph": "X",
                "pid": 17,
                "dur": 67,
                "ts": 19978045970621559,
                "cat": "Op"
            },
    '''
3. Your environment:
	+ Python version.
    'python3.5'
	+ TF version: `python -c 'import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)'`.
    'tensorflow 1.8.0'
	+ Tensorpack version: `python -c 'import tensorpack; print(tensorpack.__version__)'`.
    '0.8.6(I download the master source code and install tensorpack by python setup.py install)'

**Comment by ppwwyyxx** [comment]
Interesting.. I was able to reproduce it. Will investigate it a bit.
Typically I only run GraphProfiler for about 10 steps (as mentioned in is docs).

**Comment by ppwwyyxx** [comment]
Closing. Please track the corresponding issue in tensorflow.
